### PR TITLE
Issue 223: Revert PR 220

### LINKED ISSUE #223: Revert PR 220
**Problem description**
PR #220 introduces a dependency in Pravega Samples for which we do not have a release artifact.

**Problem location**
Pravega samples dependencies.

**Suggestions for an improvement**
Revert changes in PR #220 until Pravega Keycloak engages the regular release lifecycle.

### PR DESCRIPTION
**Pull request by RaulGracia**
**Change log description**
Reverted changes of PR #220.

**Purpose of the change**
Fixes #223.

**What the code does**
Removes dependency on Pravega Keycloak.

**How to verify it**
Build is passing. Tested Pravega samples and they work.

Signed-off-by: Raúl Gracia <raul.gracia@emc.com>

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -40,6 +40,9 @@ subprojects {
         maven {
             url "https://repository.apache.org/snapshots"
         }
+        maven {
```
Comment (fpj) [inline build.gradle]: It doesn't seem to be a big deal, but why are we adding sonatype snapshots in this PR? It is not clear how it is related.

Comment (EronWright) [reply build.gradle]: Note that this is reverting a PR, thus re-adding the sonatype snapshot repo, which the previous PR removed because it was dead code.

Comment (fpj) [reply build.gradle]: I think you are saying that we don't need this addition as it was dead gradle configuration removed by the original PR. At this point, I can go either way, removing it here or in a separate PR.

Comment (fpj) [reply build.gradle]: @RaulGracia what's your take?

Comment (RaulGracia) [reply build.gradle]: @EronWright is right, I was just reverting all the changes literally. I can keep that change that was removing that repo, not a problem.

**Comment by RaulGracia** [comment]
@fpj @EronWright I have kept the removal of sonatype on `build.gradle`. Please, have another look to see if we can merge this PR and release samples.

**Comment by EronWright** [comment]
Again please don't merge this.

**Comment by RaulGracia** [comment]
Closing in favor of #222 as Pravega Keycloak has been released.

**Pull request closed without merging**